ra-optimisation/rao-commons/src/test/java/com/farao_community/farao/rao_commons/RaoInputHelperTest.java

```diff
@@ -62,113 +46,4 @@ public void testSynchronizeNotFailSecondTime() {
             fail("AlreadySynchronizedException should not be thrown");
         }
     }
-
-    @Test
-    public void testCleanCrac() {
```

Comment:
Revert these modifications, cause it will lower test coverage (wait for removing deprecated method)

Reply:
As discussed, it wil increase code duplication